io_scene_godot/export_godot.py

```diff
@@ -156,6 +156,10 @@ def export_object(self, obj, parent_gd_node):
 
     def should_export_object(self, obj):
         """Checks if a node should be exported:"""
+        for dimension in obj.scale:
+            if dimension < 0:
+                logging.info("Negative scale is unsupported, Object '%s' was skipped", obj.name)
```

Comment:
May be better to use logging.warning(), we have a [hook](https://github.com/godotengine/godot-blender-exporter/blob/master/io_scene_godot/export_godot.py#L72) to show warning messages to the GUI.  Thanks!